examples/discordjs-typescript/src/index.ts

```diff
@@ -0,0 +1,16 @@
+import { Client, Message } from 'discord.js';
```

Comment:
```suggestion
import { Client } from 'discord.js';
```
Is this type annotation needed?

Reply:
Yes, this is why I made a separate ts example. 🙂